JongHo/app.js

```diff
@@ -42,6 +41,119 @@ app.post("/userCreate", async (request, response) => {
   response.status(201).json({ message: "userCreated" });
 });
 
+//create a post
+app.post("/postCreate", async (request, response) => {
```

Comment:
```suggestion
app.post("/post", async (request, response) => {
```
http method인 post에 이미 충분한 의미가 담겨져 있다고 생각됩니다!